src/crewai/telemetry/telemetry.py

```diff
@@ -221,6 +266,7 @@ def crew_execution_span(self, crew):
                     pkg_resources.get_distribution("crewai").version,
                 )
                 self._add_attribute(span, "crew_id", str(crew.id))
+                self._add_attribute(span, "inputs", json.dumps(inputs))
```

Comment:
This whole code only executes if the user opted-in to share the crew.

Reply:
Yup, as it's opt-in this should be okay